<launch>
    <arg name="ns" default="/odom_calibration"/>
    <arg name="odom_raw" default="/rflex/atrvjr_node/odom"/>
    <arg name="odom_calibrated" default="/odom_calibration/odom_calibration_node/odom"/>
    <arg name="parameters" default="$(find odom_calibration)/params/odom_calibration.yaml"/>

    <node ns="$(arg ns)" name="odom_calibration" pkg="odom_calibration" type="odom_calibration_node" output="screen" required="true">
        <rosparam command="load" file="$(arg parameters)" />
        <param name="odom_raw" value="$(arg odom_raw)"/>
        <param name="odom_calibrated" value="$(arg odom_calibrated)"/>
    </node>
</launch>
